src/components/QuizGame/index.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from "react";
+import { fetchQuizQuestions } from "../../components/QuizGameApi";
+import QuestionCard from "../../components/QuizGameQuestions";
+import "./index.css";
+
+const DIFFICULTY = {
+  EASY: "easy",
+  MEDIUM: "medium",
+  HARD: "hard",
+};
+const TOTAL_QUESTIONS = 10;
```

Comment:
newline above line 11, after line 11, in all code from now on too if you are declaring variables at the top level

Reply:
not fixed